store/v2/smt/store.go

```diff
@@ -24,15 +26,18 @@ type Store struct {
 	tree *smt.SparseMerkleTree
 }
 
-func NewStore(nodes, values smt.MapStore) *Store {
+// MapStore that wraps Get to raise InvalidKeyError
```

Comment:
could you update the documentation? Why we can't return InvalidKeyError directly?